McCorrections/src/LeptonWeights.cpp

```diff
@@ -114,8 +113,8 @@ double LeptonWeights::GetTriggerWeight(EventInfoBase& eventInfo) const
     const double eff_data = GetTriggerEfficiency(eventInfo, true);
     const double eff_mc = GetTriggerEfficiency(eventInfo, false);
     if(eff_mc == 0) {
-        if(eff_data != 0)
-            throw exception("Undefined trigger SF for %1%") % EventIdentifier(event);
+        if(event.lep_decayMode.at(0) >= 0 && eff_data != 0)
```

Comment:
this just hides the problem and don't performs this check for eTau and muTau. Please, identify for which exact cases it happens.